house_screen.py

```diff
@@ -50,13 +57,44 @@ def __init__(self):
         self.full_cup = self.sprite_sheet.get_image(0,864,96,96,2, config.RED)
         self.full_cup_item = scene_item.Item(config.ItemID.full_cup, pygame, self.screen, self.full_cup, self.my_rock, 150,400)
 
+        self.thought_of_watering_can = self.sprite_sheet.get_image(0, 288, 96, 96, 1, config.RED)
+        self.thought_of_ball = self.sprite_sheet.get_image(2, 288, 96, 96, 1, config.RED)
+        self.thought_of_full_cup = self.sprite_sheet.get_image(0,864,96,96,1, config.RED)
+
         self.started_game_time = pygame.time.get_ticks()
         self.not_interacted = False
         self.rock_misbehaving_time = 10 #seconds
         self.game_over = GameOver(pygame, self.screen, self.my_rock)
 
         self.pet_died = False
         self.is_rock_dirty = False
+        self.is_hungry = False
+        self.is_thirsty = False
+        self.is_sad = False
+
+    def draw_thought_bubble(self, screen, pet_rock_location, image): 
+        bubble_offset = (0, 50)  # Offset for the first bubble (above the rock)
+        bubble_x = pet_rock_location[0] + bubble_offset[0] - self.thought_bubble_radius
+        bubble_y = pet_rock_location[1] + bubble_offset[1] - self.thought_bubble_radius
+
+    # Draw the first bubble (above the rock)
+        pygame.draw.circle(screen, config.WHITE, (bubble_x + self.thought_bubble_radius, bubble_y + self.thought_bubble_radius), self.thought_bubble_radius//2)
+
+    # Update the bubble_offset for the second bubble (below the first bubble)
+        bubble_offset = (0, -50)  
+        bubble_x = pet_rock_location[0] + bubble_offset[0] - self.thought_bubble_radius
+        bubble_y = pet_rock_location[1] + bubble_offset[1] - self.thought_bubble_radius
+
+    # Draw the second bubble (below the first bubble)
+        pygame.draw.circle(screen, config.WHITE, (bubble_x + self.thought_bubble_radius, bubble_y + self.thought_bubble_radius), self.thought_bubble_radius)
+
+    # Draw the broccoli sprite inside the main thought bubble
+        item_x = bubble_x + (self.thought_bubble_radius - self.broccoli_item.rect.width // 2)
+        item_y = bubble_y + (self.thought_bubble_radius - self.broccoli_item.rect.height // 2)
+        screen.blit(image, (item_x, item_y))
+
```

Comment:
🤔 To have the broccoli inside the thought bubble, would it have been easier for me to adjust the `BarIcon` class I worked on to return an icon so that it would be easier to display and specific icon inside the bubblee and make it easier to change the thoughts dynamically?

Reply:
Probably, I think that the rectangle size is about the same as the others so I should be able to change it to any of them or just set a value that could work for all of them. I forgot to swap that out when I was changing the hunger thought to the rest, but I'll do that before the final pull.

Reply:
👍🏽 Sounds good